### PR TITLE
0.1.0 release

### LINKED ISSUE #53: Release 0.1.0
@pandafy I think that after the API is done (#48) we can issue the first release of the module and publish it to pypi.

This should push us to make everything prettier (eg: eliminate all "TODO" placeholders from the readme) and will help the rest of the OpenWISP community to take this effort more seriously.

Once #48 is merged, can you please prepare a summary of the most relevant features we added in the change log (see `CHANGES.rst` in the other modules) please? And set the version to 0.1.0 final. I'll then proceed to publish it on pypi and add you as a maintainer (create an account there if you don't have one yet).

Thanks in advance.

### PR DESCRIPTION
**Pull request by pandafy**
Closes #53

### CODE REVIEW

**Comment by pandafy** [comment]
> I forgot that we need to release openwisp-users as well and we need to depend on the latest version.

openwisp-users dependency was upgraded in https://github.com/openwisp/openwisp-notifications/commit/f90ae6809ada85baa5edbb3089aadc81944da831.  See [requirements.txt](https://github.com/openwisp/openwisp-notifications/commit/f90ae6809ada85baa5edbb3089aadc81944da831#diff-b4ef698db8ca845e5845c4618278f29a). 

I have opened a PR in openwisp-users for release: https://github.com/openwisp/openwisp-users/pull/131

@nemesisdesign  I am not able to activate require,io monitoring for this repository. Maybe permission issues, can you please activate it? :smile:

File: setup.py [hunk 1]
```diff
@@ -42,12 +42,12 @@ def get_install_requires():
     name='openwisp-notifications',
     version=get_version(),
     license='GPL3',
-    author='Gagan Deep',
```
Comment (NoumbissiValere) [inline setup.py:45]: I think this `author` and `author_email` should be maintained since it is your work. :smile:

**Comment by NoumbissiValere** [comment]
Please also rebase it with master

File: setup.py (hunk 1 above)
Comment (nemesisdesign) [reply setup.py:45]: :+1:

File: setup.py [hunk 2]
```diff
@@ -42,12 +42,12 @@ def get_install_requires():
...
-    author_email='the.one.above.all.titan@gmail.com',
+    author='Federico Capoano',
+    author_email='federico.capoano@gmail.com',
     description='OpenWISP 2 Notification Module',
```
Comment (nemesisdesign) [inline setup.py:47]: Let's use `Notifications module of OpenWISP`

File: CHANGES.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Changelog
+=========
+
+Version 0.1.0 [2020-01-03]
```
Comment (nemesisdesign) [inline CHANGES.rst:4]: the date needs to be updated

**Pull request merged**